MiniApp/Classes/core/RealMiniApp.swift

```diff
@@ -212,6 +227,54 @@ internal class RealMiniApp {
     func cleanUpKeychain() {
         self.miniAppStatus.removeUnusedCustomPermissions()
     }
+
+    /// Method to check if all the required permissions mentioned in the manifest.json is agreed by the user.
+    /// - Parameters:
+    ///   - appId: MiniApp ID
+    ///   - versionId: Specific VersionID of a MiniApp
+    ///   - completionHandler: Handler that returns whether user agreed to required permissions or not.
+    func isRequiredPermissionsAllowed(appId: String, versionId: String, completionHandler: @escaping (Result<Bool, MASDKError>) -> Void) {
```

Comment:
The `RealMiniApp` class is intended to just be the place where the various components are connected. But it feels like it's turning into a "god class" which has too much functionality. Is there any way you could refactor some of this permissions functionality into another class? 

Sorry, I don't really have a good suggestion on this. But basically, I feel like `RealMiniApp` is getting too large. Maybe it's something we can add as "technical debt" and address in another PR?

Reply:
Yeah, I felt the same. We can add extensions or move few functions into a separate class. But don't want to do this close to the release. Will work on this after the release

Reply:
I think using an extension would not be different from having this big class. We should move all the methods that don't need to be instanced to class methods in a dedicated tools class.

Reply:
Yes, we can. But I wanted to avoid any refactoring now, especially when we are this close to release.